src/core/qgsbrowsermodel.cpp

```diff
@@ -101,6 +101,14 @@ void QgsBrowserModel::addRootItems()
     mRootItems << mFavorites;
   }
 
+  // add bookmarks
+  mBookmarks = new QgsBookmarksItem( nullptr, tr( "Bookmarks" ), QgsApplication::bookmarkManager(), QgsProject::instance()->bookmarkManager() );
```

Comment:
Could you move this to app and implement as a provider? It'll be cleaner and will avoid the QgsProject::instance call from core